.github/workflows/release-napi.yml

```diff
@@ -131,6 +134,12 @@ jobs:
           cache: "npm"
           cache-dependency-path: crates/bitwarden-napi/package-lock.json
 
+      - name: Retrieve schemas
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: schemas.ts
+          path: ${{ github.workspace }}/crates/bitwarden-napi/src-ts/bitwarden_client/
```

Comment:
I think we would already have these generated at this point? We download artifacts from the `build-napi` workflow.

Reply:
Hmm that's likely the case yeah, though I'm not sure how to test the release workflows. I removed it for now

Reply:
I think this needs to be solved before this can be merged. I believe a good starting point is just to try and download them from the generate_schema workflow but not trigger it and have devops verify it looks good.

Reply:
I believe we have to update the code for the release workflow so that it downloads the already built schema.ts. Since it's a required part of the build process.

Reply:
Yes. I think that should work.

Reply:
Done in 28ab623e5f6bfadc3697bd550ac316cc5eab429c

Reply:
I think you need to specify some more fields to pick a workflow etc.